.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+   // Use IntelliSense to find out which attributes exist for C# debugging
```

Comment:
Why are these lines rendered differently on github? Did you use special characters?

Reply:
JSON isn't technically allowed to have comments :(

VS Code puts those in there to help though

Reply:
makes sense but are you sure that we don't get into trouble parsing the json? I just fund a possible workaround: https://www.quora.com/How-do-I-write-comments-inside-a-JSON-document

Reply:
haha nice. I can update to do that